src/context/intermediate/Intermediate.tsx

```diff
@@ -173,6 +176,10 @@ export default function Intermediate(props: Props) {
 
                 return true;
             },
+            confirmSessionsDelete: (confirmDelete: () => void) => openScreen({
+                id: "sessions",
+                confirmDelete: confirmDelete
+            }),
```

Comment:
For "screens" that are mentioned only once elsewhere, I prefer that everything goes through `openScreen` as to not let this get out of control.

This piece of code would need to be removed and the contents moved as detailed in next comment.